submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+## Best of its genre - Nanogamer7
+
+We've all heard of Factorios insanely [good ratings](https://steamdb.info/stats/gameratings/) - consistently rising to second place after Portal 2 again and again.[^1] Obviously many others will try to imitate this success in the automation genre, but where did Factorio get inspired from? That's what I'll be trying to answer in this first article of hopefully a multi-part series.
+
+### Automation defined
+
+The first difficult question we'll have to answer though, would be what automation even is. In Cities:Skylines for example you are building your city, which then (theoretically) functions autonomously. The problem here is, your job isn't to automate - the cars drive themselves, once you build roads - but to manage it. In Minecraft on the other hand, you are the one to make the automation even possible, just with pistons and hoppers, instead of inserters and belts. Comparing these two examples, I think you can loosely define the automation genre as *"making stuff that makes stuff"*.
+
+However that definition is still a bit vague: From what I know about e.g. Dwarf Fortress, you could consider enabling dwarfs to *make stuff* as falling under that definition. But let's not get to far from Factorio here, maybe someone else will explore this further in the future. *\*wink wink\**
+
+### There's a mod for it
+
+Speaking of Minecraft, it's popularity boom in 2013 and with it one of the largest modding communities ever,[^2][^3] it was most definitely the first introduction to technical gameplay - and to some degree even automation - to many players, especially younger ones. But that got me thinking: With mods in the likes of Buildcraft, IndustrialCraft, and Redpower featuring a very heavy focus on industrial machinery and automation, and release dates well before Factorio's development even started,[^4] could that have been an inspiration for Factorio?
+
+Wube's online presents, while there, is spread out across the forums, reddit, discord, and a few more, which makes it difficult to search for anything, really. But after some clever searching and sifting through a few interviews, I finally found  something: An [old forum thread](https://forums.factorio.com/viewtopic.php?f=5&t=3026) from March 2014, confirming Buildcraft (or IndustrialCraft, according to a different interview) as the main inspiration to Factorio.[^5][^6] While I had hoped to find an answer, I for sure wasn't expecting to find such a direct one. Having that out of the way, though, we can definitely speculate a bit more on brought Factorio to were it's at today.
```

Comment:
Maybe start with "i went researching.. but hard because multiple". Otherwise it seems like youre randomly telling us that Wube has a lot of social media.